<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFrame</class>
 <widget class="QMainWindow" name="CFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CFrame</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_Listas">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>891</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_disponibles">
     <attribute name="title">
      <string>Disponibles</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_disponibles">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>841</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_prioridades">
     <attribute name="title">
      <string>Prioridades</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_priority_type">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>861</width>
        <height>341</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_PAlta">
       <attribute name="title">
        <string>Alta</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_PAlta">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>831</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_PMedia">
       <attribute name="title">
        <string>Media</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_PMedia">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>831</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_PBaja">
       <attribute name="title">
        <string>Baja</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_PBaja">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>831</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Acciones">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>751</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Acciones</string>
    </property>
    <widget class="QPushButton" name="pBtn_Mover">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Mover</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Mover">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_Pos_aMover">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Posición a Mover:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtn_Agregar_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar Nueva Casa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtn_Vender">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Vender Casa</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Vender">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>80</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_Pos_aVender">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>80</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Posición a Vender:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_Title_tabla">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif Ahom</family>
      <pointsize>38</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Casas Disponibles</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="Julio">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>590</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Indira">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>590</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>690</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Edwin Espino</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>690</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Julio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>690</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Indira Mariel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>690</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Josué Maldonado</string>
    </property>
   </widget>
   <widget class="QLabel" name="Edwin">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>590</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Edwin.jpeg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Josue">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>580</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
